styles/style.css

```diff
@@ -0,0 +1,573 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 60%;
+}
+
+body {
+  font-weight: 300;
+  line-height: 1.6;
+  position: relative;
+  width: 100%;
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #f4f5f7;
+}
+
+.color-white {
+  color: white;
+}
+
+.color-orangered {
+  color: #f61602;
+}
+
+.pointer {
+  cursor: pointer;
+}
+
+/* -------------- NABVAR ---------------*/
+
+.nav-container {
+  position: relative;
+}
+
+.navbar-big {
+  transform: translateY(-5rem);
+  z-index: 15;
+}
+
+.sticky-navbar {
+  position: sticky;
+  top: 0;
+  width: 100%;
+  left: 0;
+  z-index: 9;
+  height: 5rem !important;
+}
```

Comment:
## No `!important` overusing[1]

The !important rule was conscripted in the mid-to-late 90’s to help web designers and developers override the rules of their style sheet. 

- [ ] Remove all `!important` rules in your entire CSS file

<hr>

**Note:** Check all this entire page and fix them

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

Reply:
Thank yo very much for your review, I already made the changes you requested for.

Happy coding.